.github/etc/README

```diff
@@ -0,0 +1,2 @@
+
+This is used by both Azure and GHA and they use different values for macOS, hence two entries.
```

Comment:
```suggestion
The ccache settings in this directory are required by R builds on both Azure and
GHA and they use different values for macOS, hence two entries.
```
I know I am being pedantic, but otherwise someone may not know why any of these are needed.

Reply:
As per the discussion above, this change anyway as we no longer prepare for R bindings on Azure so the comment will go.  I will probably get to that in a few minutes.

Reply:
This file is going to be deleted in the next commit.

(I do like editorial commits / suggestion though!)

Reply:
Deleted in c4b47ac